testpar/t_vfd.c

```diff
@@ -0,0 +1,4057 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * Copyright by the Board of Trustees of the University of Illinois.         *
```

Comment:
The header comment below mentions 2015 as the date, but if this file is new it shouldn't have the U of I board in the copyright.